uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -1609,33 +1610,46 @@ sub my_trim {
 
 =pod
 
-=head3 isDicom(@files_list)
+=head3 isDicomImage(@files_list)
 
-This method checks whether the list of files given as an argument is of type DICOM.
+This method checks whether the files given as an argument are of type DICOM. All
+DICOM image files will be returned in the array @image_files. Non-DICOM image or
+DICOM scanner reports that are not images will be returned in the array
+@non_image_files.
 
 INPUT: array with full path to the DICOM files
 
 RETURNS:
-  - @dicom_files    : array with the list of DICOM medical imaging data files
-  - @non_dicom_files: array with the list of non-DICOM files
+  - @image_files    : array with the list of DICOM medical imaging data images
+  - @non_image_files: array with the list of non-DICOM images
 
 =cut
 
-sub isDicom {
+sub isDicomImage {
     my (@files_list) = @_;
 
     my $cmd = "file " . join(' ', @files_list);
```

Comment:
I think this command could fail for large file lists (we could exceed the maximum length a Unix command can have). Maybe we could change this to:
```perl
my $cmd = "ls @files_list | xargs file";
```
just to be safe. What do you think?

Reply:
good point!